lib/addons/src/types.ts

```diff
@@ -11,3 +13,83 @@ export type Types = types | string;
 export function isSupportedType(type: Types): boolean {
   return !!Object.values(types).find(typeVal => typeVal === type);
 }
+
+export interface Parameters {
+  fileName?: string;
+  options?: OptionsParameter;
+  [key: string]: any;
```

Comment:
do we really want to allow this wildcard?
addons can argument this interface to add their own params.

the same question applies to `StoryContext`

Reply:
Yeah these are extensible by design.

Reply:
i think we missunderstand.

```ts
export interface Parameters {
  fileName?: string;
  options?: OptionsParameter;
}
```
is just as extensible when using interface augmentation, it just doesnt allow untyped params (because no index signature). see https://www.typescriptlang.org/docs/handbook/declaration-merging.html#merging-interfaces